package.json

```diff
@@ -85,6 +85,7 @@
     "mocha": "^6.2.2",
     "nodemon": "^2.0.2",
     "nyc": "^15.0.0",
+    "polendina": "^1.0.0",
```

Comment:
instead of using npx here, I've just included polendina in our dev deps. This means we get caching for it in GH actions.